app/controllers/work_packages/auto_completes_controller.rb

```diff
@@ -54,15 +54,15 @@ def index
                       .limit(10)
 
     respond_to do |format|
-      format.html do render layout: false end
-      format.any(:xml, :json) { render request.format.to_sym => wp_hashes_with_string }
+      format.html { render layout: false }
```

Comment:
This seems to be the only change where I disagree:

We chose to distinguish between `do ... end` and `{ ... }` based on semantics. Best match on the internet might be [this one](https://github.com/bbatsov/ruby-style-guide/issues/162), which has some sources itself.

Basically: When the content of the block is about the side-effect, use `do ... end`, when it is about the return-value use `{ ... }`.